### DISCUSSION TITLE
Strict Enforcement of Anti-Spam Rules

### DISCUSSION THREAD

**Discussion by StrangeWill** [devanooga/devanooga-meta#69] [Rules & Moderation]
So far I've been pretty aggressive about fighting accounts that appear to be:

- Not local
- Swooping in, posting a job posting
- Not responding to anyone's replies (usually about "this is the wrong channel")

These posts all seem to have similar attributes: 

- They come from the wiki list that we're on (https://github.com/ladyleet/tech-community-slacks)
- They commonly seem to be outsourcing firms, _maybe_ recruiters (they never seem to represent the company they're posting for)
- Or they're outright scams (eg: upwork account sharing for a cut of the profit)

I've been outright banning immediately, is this too aggressive? Do we care? I figure if they care they can open a dispute, but that would require them reading our site.

**Comment by brb3** [comment]
The M.O. for them seems to be to cast a wide net and see what sticks. Interacting with the accounts you are describing normally yields nothing -- they are either aloof or unresponsive.

I approve of banning immediately with this behavior pattern.
